.vib/clickhouse/vib-publish.json

```diff
@@ -0,0 +1,81 @@
+{
+  "phases": {
+    "package": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/clickhouse"
+        }
+      },
+      "actions": [
+        {
+          "action_id": "helm-package"
+        },
+        {
+          "action_id": "helm-lint"
+        }
+      ]
+    },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/clickhouse"
+        },
+        "runtime_parameters": "YXV0aDoKICBwYXNzd29yZDogYml0bmFtaTEyMzQKCnNlcnZpY2U6CiAgdHlwZTogTG9hZEJhbGFuY2VyCiAgcG9ydHM6CiAgICBodHRwOiA4MAo=",
```

Comment:
When selecting the runtime parameters, we usually go with the following mindset: "read the `values.yaml` file and look for relevant values that are set by default. Select those params and write them down. Change their default values to see if the chart correctly handles them"

That said, here is the current `runtime_parameters`:

```
auth:
  password: bitnami1234

service:
  type: LoadBalancer
  ports:
    http: 80
```

Additionally, I would add the following:

- Change the default value for `auth.username`
- Change the containerPorts values to `current_value + 1`
- Use the securityContext to set a user and fsGroup != 1001 (usually, we go with 1002)
- Change the rest of the ports of the service to `current_value + 1`
- Explicitly add an entry for `serviceAccount` with `create=true` and `automountServiceAccountToken=true`, as it is the default value used by default and we are testing it in GOSS.

Reply:
I'm changing it, but I have a concern with this. In the past, we were changing so many values in our tests that, in the end, we weren't testing our charts with the default `helm install` command. I don't remember the chart, but we got support cases from users saying that the chart was not installable with `helm install` (which was a bit embarrasing)

Reply:
>I'm changing it, but I have a concern with this. In the past, we were changing so many values in our tests that, in the end, we weren't testing our charts with the default helm install command. 

But we are actually not adding any extra installation params here 😁 . We install the chart with the default params in `values.yaml`, but we change their value. 

This way, not only do we test that the "default" installation works as expected, but that other values do not break the chart (e.g. changing the user name from "default" to "testBitnami" should not break the chart).

Reply:
That's debatable and would depend on how the helm chart is developed. For example, we're not testing whether the random generation of the password is working, for example. I would need to recall what the issue was back then. Anyways, I updated the PR with the requested params.